source/views/student-orgs/list.js

```diff
@@ -83,9 +83,19 @@ export class StudentOrgsView extends React.Component {
           $groupableName: head(startCase(sortableName)),
         }
       })
+      // All orgs
       let sorted = sortBy(withSortableNames, '$sortableName')
       let grouped = groupBy(sorted, '$groupableName')
-      this.setState({orgs: grouped})
+      // New orgs
+      let newOrgs = sorted.filter(org => org.newOrg).map(item => {
+        return {
+          ...item,
+          $groupableName: 'New',
+        }
+      })
+      let newOrgsGrouped = groupBy(newOrgs, '$groupableName')
+      let combined = {...newOrgsGrouped, ...grouped}
```

Comment:
Please inline this into the setState call (setState({orgs: {...newOrgs, ...allOrgs}})